src/main/java/org/dmfs/oauth2/client/http/responsehandlers/TokenResponseHandler.java

```diff
@@ -53,11 +53,11 @@ public TokenResponseHandler(OAuth2Scope scope)
     @Override
     public OAuth2AccessToken handleResponse(HttpResponse response) throws IOException, ProtocolError, ProtocolException
     {
-        if (!APPLICATION_JSON.equals(response.responseEntity().contentType()))
+        if (!APPLICATION_JSON.equals(response.responseEntity().contentType().value()))
```

Comment:
The check and error handling of missing content type could be explicit here, I guess, but I suppose it's not very important, since that will almost never happen, right?
(Same for `TokenErrorResponseHandler`)

Reply:
Right and even if it does, there is no need to be explicit IMO.